src/text-field/text-field.tsx

```diff
@@ -9,7 +9,7 @@ type TextFieldType = 'email' | 'search' | 'tel' | 'text' | 'url'
 
 interface TextFieldProps
     extends Omit<FieldComponentProps<HTMLInputElement>, 'type'>,
```

Comment:
```suggestion
    extends Omit<FieldComponentProps<HTMLInputElement>, 'type' | 'supportsStartAndEndSlots'>,
```

Reply:
For whatever reason, this doesn't remove it from Storybook.

Reply:
Mmm it does remove it for me (after restarting Storybook).

Reply:
Which it does. Stupid Storybook.